docker/Jenkinsfile-build-docker

```diff
@@ -19,7 +19,7 @@ properties([
 
 IMAGE_NAME_PREFIX = 'harbor.h2o.ai/opsh2oai/h2o-3'
 
-JDK_VERSIONS = ['8', '10', '11', '12', '13', '14']
+JDK_VERSIONS = ['8', '10', '11', '12', '13', '14', 15]
```

Comment:
how many images this added? probably a few, we really need to take a hard look at that

Reply:
This is the most important discussion. Yes, I agree. Should I skip building the images and not add Java 15 to smoke tests ? I can test the checks using plain JUnit tests I believe.

Or, add 15 in this PR and in another PR let's optimize this to the oldest and newest supported only ?